src/couldBeSharedComponents/LayoutPanels/LayoutPanels.tsx

```diff
@@ -140,18 +140,24 @@ export const LayoutPanels = ({
     }
   };
 
+  useEffect(
+    function syncContentVisibilityToPanelVisibility() {
+      // Ensure panel content is visible to assistive technology and tabbing before the panels are shown
+      // We only sync when panels are showing. The transition end handler functions are responsible
+      // for hiding content when panels are closed (we don't hide content before the transition ends as that would be jarring).
+      if (isLeftPanelOpenToUse) {
+        setIsLeftPanelContentShowing(true);
+      }
+      if (isRightPanelOpenToUse) {
+        setIsRightPanelContentShowing(true);
+      }
+    },
+    [isLeftPanelOpenToUse, isRightPanelOpenToUse],
+  );
```

Comment:
Controlling state externally means I cant rely on controlling this in the internal button click handler! Yay for tests revealing all the things keeping stuff from being robust.